packages/gatsby/lib/schema/build-node-types.js

```diff
@@ -1,178 +1,167 @@
 // @flow
-const _ = require(`lodash`)
-const {
+import _ from 'lodash'
```

Comment:
btw, it's not a big deal but I've been using just require in node.js since it's not supported there and I write enough raw (haha) node.js code that it trips me up otherwise using imports.